### PR TITLE
Addded new stuff to sooper dooper powershell script

### PR DESCRIPTION
**Pull request by gartee-john-PFG**
2 new functions and something else
@DavisLeonPFG , look at this hard

### CODE REVIEW

File: Nicki.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Awesomesauce 1
+Awesomesauce 2
+Awesomesauce 3
```
Comment (SusmanNickiPFG) [inline Nicki.txt:3]: Why yes, this is awesome sauce!

**Review by Jason-Wilson-PFG — COMMENTED** [review]
This looks super dooper

File: gartee.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+line 1
+line 2
+line 3
```
Comment (SusmanNickiPFG) [inline gartee.txt]: Kinda boring

File: Nicki.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+Awesomesauce 1
+Awesomesauce 2
```
Comment (SusmanNickiPFG) [inline Nicki.txt:2]: This is fantastic!!

File: Nicki.txt (hunk 1 above)
Comment (PooleSamanthaPFG) [reply Nicki.txt:3]: cool beans!

File: Nicki.txt (hunk 3 above)
Comment (SusmanNickiPFG) [reply Nicki.txt:2]: On second thought, maybe we should review this. Is it awesomesauce or should it be unicorn?

Comment (SusmanNickiPFG) [reply Nicki.txt:2]: Wait, maybe not!

File: gartee.txt [hunk 4]
```diff
@@ -0,0 +1,3 @@
+line 1
+line 2
```
Comment (SusmanNickiPFG) [inline gartee.txt:2]: I think we should add a second line 2 here

**Review by SusmanNickiPFG — CHANGES_REQUESTED** [review]
I need some changes please!!

File: gartee.txt [hunk 5]
```diff
@@ -1,3 +1,4 @@
-line 1
+line 1 
 line 2
-line 3
\ No newline at end of file
+added at nicki's request
+line 3 exciting
```
Comment (SusmanNickiPFG) [inline gartee.txt:4]: Much better!

**Review by SusmanNickiPFG — APPROVED** [review]
Much better, great job!

**Review by DaltonJayPFG — APPROVED** [review]
Nice change.

File: Nicki.txt [hunk 6]
```diff
@@ -0,0 +1,3 @@
+Awesomesauce 1
```
Comment (PooleSamanthaPFG) [inline Nicki.txt]: This should say Cool Beans

**Review by PooleSamanthaPFG — CHANGES_REQUESTED** [review]

File: Nicki.txt (hunk 6 above)
Comment (gartee-john-PFG) [reply Nicki.txt]: fixed

**Review by PooleSamanthaPFG — APPROVED** [review]
excellent work John!

**Pull request merged**